src/webapp_vite/src/ArmView/ArmAngleControl.tsx

```diff
@@ -133,6 +139,21 @@ export function ArmControl({
         );
     }
 
+    const handleCircleClick = (event: React.MouseEvent | React.TouchEvent) => {
+        const svg = svgRef.current;
+        if (!svg) return;
+        const isTouch = isTouchEvent(event);
+
+        const clientX = isTouch ? event.touches[0].clientX : event.clientX;
+        const clientY = isTouch ? event.touches[0].clientY : event.clientY;
+        const rect = svgRef.current.getBoundingClientRect();
+        const centerX = rect.left + CIRCLE_CENTER.x / 2;
+        const centerY = rect.top + CIRCLE_CENTER.y;
+
+        const angle = findAngle(centerX, centerY, clientX, clientY);
```

Comment:
This looks a lot like the handleMouseMove handler in the useEffect at top.  Maybe dry it up